README.md

```diff
@@ -25,8 +25,8 @@ Btw it's not another Electron app😉
 
 ## 🌃 Features
 
-- 🚫 No ads, thanks to the use of public & free Spotify and YT Music APIs¹
-- ⬇️ Downloadable tracks
+- 🚫 Advertisement free, thanks to the use of public & free Spotify and YT Music APIs¹
```

Comment:
I think this is the only controversial change. "No ads" is more succinct than your proposition in my opinion. The rest is good I think.

Reply:
Okay yeah, I agree. I'll look into it tomorrow :)